src/impact-dashboard/ImpactDashboard.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components";
+
+const Container = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const FormContainer = styled.div`
+  border-right: 1px solid black;
+  flex: 1 0 auto;
+  width: 350px;
+`;
+
+const ChartsContainer = styled.div`
+  flex: 2 0 auto;
+  width: 350px;
+  margin: 0 15px;
+`;
```

Comment:
Thank you for using flexbox! I think this is the easier and more maintainable responsiveness solution for components like this.